composer.json

```diff
@@ -1,26 +0,0 @@
-{
-    "name": "phpro/symfony-conventions",
-    "description": "The basic style rules",
-    "type": "library",
-    "license": "propriatary",
-    "minimum-stability": "dev",
-    "prefer-stable": true,
-    "require": {
-        "brianium/paratest": "*",
-        "friendsofphp/php-cs-fixer": "^2.16",
-        "nikic/php-parser": "^4.3",
-        "php-parallel-lint/php-parallel-lint": "^1.2",
-        "phpro/grumphp": "^1.3",
```

Comment:
I'dd say use `phpro/grumphp-shim`. Been using it for quite a while now and works good!